bibtex-actions.el

```diff
@@ -115,6 +117,23 @@ manager like Zotero or JabRef."
   :group 'bibtex-actions
   :type '(repeat function))
 
+(defvar bibtex-actions-field-map
+  '(("date" "year" "issued")
+    ("=key=" "id")
+    ("=type=" "type")
+    ("tags" "keywords" "keyword")
+    ("booktitle" "container-title")
+    ("journaltitle" "journal" "container-title")
+    ("number" "issue")))
+
+(defvar bibtex-actions-field-mapping
```

Comment:
Need to make a decision on which approach to take on mapping.

Reply:
I'm actually thinking seriously about standardizing on the csl names, as they should work better for this use case.